resources/assets/js/pdf.pdfmake.js

```diff
@@ -1030,13 +1037,15 @@ NINJA.invoiceLines = function(invoice, isSecondTable) {
                 value = lineTotal;
             }
 
-            if (j == 0) {
-                styles.push('firstColumn');
+            if (k == 0) {
+                styles.unshift('firstColumn');
```

Comment:
According to the pdfmake examples the last style item should override the values of the items before. However in this case it did not work for me. Making 'firstColumn' the first item works.